src/Mono.Android/Android.Runtime/AndroidRuntime.cs

```diff
@@ -15,8 +15,10 @@ namespace Android.Runtime {
 
 	class AndroidRuntime : JniRuntime {
 
-		internal AndroidRuntime (IntPtr jnienv, IntPtr vm, bool allocNewObjectSupported, IntPtr classLoader, IntPtr classLoader_loadClass)
-			: base (new AndroidRuntimeOptions (jnienv, vm, allocNewObjectSupported, classLoader, classLoader_loadClass))
+		internal AndroidRuntime (IntPtr jnienv, IntPtr vm, bool allocNewObjectSupported, IntPtr classLoader, IntPtr classLoader_loadClass,
+		                         bool jniAddNativeMethodRegistrationAttributePresent)
```

Comment:
Please, let's *not* align parameters to `(`.  (I know Emacs likes that.  Emacs is wrong. ;-)

Rationale: If (when?) this type is renamed, it means that either all of these will be *mis*-indented, or they'll need to be re-indented so that things look proper.

Instead, it should be indented two tabstops, and/or all parameters should be done on a newline, each one being two tabstops:

```csharp
internal AndroidRuntime (IntPtr jnienv,
		IntPtr vm,
		bool allocNewObjectSupported,
		IntPtr classLoader,
		IntPtr classLoader_loadClass,
		bool jniAddNativeMethodRegistrationAttributePresent)
{
}
```

Reply:
Emacs isn't wrong, but I don't care enough to defend the position :) Changes applied (in 2 more places in addition to this)